### PR TITLE
add delegate for collectionview batch update completion

### PR DESCRIPTION
**Pull request by anneWe**

### CODE REVIEW

**Comment by jflinter** [comment]
This seems reasonable, @anneWe, but would you mind adding tests for this functionality please?

**Comment by anneWe** [comment]
sure, i will update my PR with tests.

File: Dwifft/Dwifft+UIKit.swift [hunk 1]
```diff
@@ -83,7 +90,10 @@ open class CollectionViewDiffCalculator<T: Equatable> {
 
                     self.collectionView?.insertItems(at: insertionIndexPaths)
                     self.collectionView?.deleteItems(at: deletionIndexPaths)
-                }, completion: nil)
+                
+                }, completion: { (completion) in
```
Comment (sahandnayebaziz) [inline Dwifft/Dwifft+UIKit.swift]: @anneWe On this line, the boolean parameter `completion` isn't being used. Can we change the signature to `_ in` to make it more obvious that it's not used?

[More information](https://developer.apple.com/reference/uikit/uicollectionview/1618045-performbatchupdates) on the purpose of this parameter.

**Comment by jflinter** [comment]
Closing due to lack of activity

**Pull request closed without merging**